index.bs

```diff
@@ -257,8 +257,13 @@ Try to construct a table with the columns of the properties of <var>tabularData<
 
 <h4 id="trace" oldids="trace-data,dom-console-trace" method for="console">trace(...<var>data</var>)</h4>
 
-Performs Logger("log", <var>data</var>) and prints a snapshot of the call-stack from where the method was called.
-<p class="note">The identifier of a function printed in a stack trace is implementation-dependant. It is also not guaranteed to be the same identifier that would be seen in <var>new Error().stack</var></p>
+<emu-alg>
+  1. Let _trace_ be some implementation-specific, potentially-interactive representation of the callstack from where this method was called.
+  1. Optionally let _formattedData_ be the result of Formatter(<var>data</var>), and incorporate _formattedData_ as a label for _trace_.
+  1. Perform Printer("log", _trace_).
```

Comment:
Printer() should take a list as its second parameter, so do `« _trace_ »`. There is an existing usage where it does not that is a bug; I will open a separate issue.